wallet/wallet-rpc-lib/src/rpc/interface.rs

```diff
@@ -154,6 +154,24 @@ trait WalletRpc {
         options: TransactionOptions,
     ) -> rpc::RpcResult<NewTransaction>;
 
+    #[method(name = "address_sweep_spendable")]
+    async fn sweep_addresses(
+        &self,
+        account: AccountArg,
+        destination_address: String,
+        from_addresses: Vec<String>,
+        options: TransactionOptions,
+    ) -> rpc::RpcResult<NewTransaction>;
+
+    #[method(name = "staking_sweep_delegation")]
+    async fn sweep_delegation(
+        &self,
+        account: AccountArg,
+        destination_address: String,
+        delegation_id: String,
```

Comment:
I'm not sure why we need separate function for that. Can we extend `withdraw_from_delegation` with something like enum {Max, Amount(Amount)}; ? 

Imho separate function would make sense if we withdraw from all pool's delegations or all account's delegations. 

Just a suggestion

Reply:
It might work... we'll need to change the functions' parametrization so that it can take different input data with different meanings, because "max" is something that can be entered and it's not a number. Don't forget that fee calculation has to be done on the fly. It's not such a bad idea, but I think having "sweep" functions is just easier to understand, since it's available in other wallets for other blockchains.